### ISSUE TITLE
Firebase support

### ISSUE THREAD

**Issue by hackerunet** [node-modli/modli#11]
Hello, I was wondering if is possible to implement a firebase adapter with the available classes on this library?, where should i start with? 
Thanks

**Comment by Fluidbyte** [comment]
Yes, it's completely possible. You can just check out one of the [other adapters](https://github.com/node-modli) (`modli-nedb` is probably the easiest to just fork) then build around the methods that are in that.
